monai/apps/nnunet/nnunetv2_runner.py

```diff
@@ -495,65 +496,61 @@ def train_single_model(self, config: Any, fold: int, gpu_id: tuple | list | int
             fold: fold of the 5-fold cross-validation. Should be an int between 0 and 4.
             gpu_id: an integer to select the device to use, or a tuple/list of GPU device indices used for multi-GPU
                 training (e.g., (0,1)). Default: 0.
-        from nnunetv2.run.run_training import run_training
             kwargs: this optional parameter allows you to specify additional arguments in
                 ``nnunetv2.run.run_training.run_training``. Currently supported args are
```

Comment:
Please change ``nnunetv2.run.run_training.run_training`` to  ``nnunetv2.run.run_training.run_training_entry``. Thanks.